code/game/objects/items/weapons/implants/implantcase.dm

```diff
@@ -121,3 +121,15 @@
 		src.imp = new /obj/item/weapon/implant/death_alarm( src )
 		..()
 		return
+
+//Pax(Peace) Implant Case
+/obj/item/weapon/implantcase/peace
+	name = "Glass Case- 'Pax'"
+	desc = "A case containing a peace implant."
+	icon = 'icons/obj/items.dmi'
+	icon_state = "implantcase-b"
+
+	New()
```

Comment:
this is a relative path, also known as the devil. It's one of the few things we have guidelines for https://github.com/vgstation-coders/vgstation13/wiki/Coding-standards-for-vgstation13#absolute-pathing